multicluster/config/samples/clusterset_init/leader-access-token-template.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: leader-access-token
+  name: default-member-token
   namespace: antrea-multicluster
   annotations:
     kubernetes.io/service-account.name: antrea-mc-member-access-sa
```

Comment:
@luolanzone : it is not very important, but I wanted to rename the SA to "antrea-mc-member-default" to be consistent with the default token name, but did not find out how to change it. Could you let me know how to change the name?

Another question - `make manifests` in my dev VM updates many YAMLs. Is that due to my env, or it is possible YAMLs merged to the repo are not update to date?

Reply:
You need to replace `member-access-sa` to `member-default` on two files `member_cluster_rolebinding.yaml` and  `member_cluster_serviceaccount.yaml` in the folder `multicluster/configs/overlays/leader-ns/`. You  probably need to clean up the file on `multicluster/bin/controller-gen` and rerun `make manifests`. It has been upgrade to v0.9.

Reply:
Ok. Let us consider changing with a follow up PR.